content/reference/cli/cli.md

```diff
@@ -49,7 +49,7 @@ Valid commands for dolt are
                 docs - Commands for working with Dolt documents.
                stash - Stash the changes in a dirty working directory away.
              profile - Manage dolt profiles for CLI global options.
-          query-diff - description
+          query-diff - Calculates table diff between two queries.
```

Comment:
Was this file updated with the output of `dolt dump-docs`? I still see the old `description` when I run that. And it looks like there should be a new line added for `reflog`, too. 

It's better to run `dolt dump-docs` to generate this file, otherwise any changes will be overwritten by the next person who uses that to generate this file. Could you also check out the README in this repo and see if we need more instructions on that? I can't remember how well we documented that process.